full/intro-clustering-R-tidymodels/README.md

```diff
@@ -0,0 +1,130 @@
+
+
+## Prerequisites
+
+To get you up and running and writing R code in no time, we have containarized this workshop such that you have a ready out of the box R coding environment.
+
+### Setting up the development container
+
+A **development container** is a running [Docker](https://www.docker.com) container with a well-defined tool/runtime stack and its prerequisites. You can try out development containers with **[GitHub Codespaces](https://github.com/features/codespaces)** or **[Visual Studio Code Remote - Containers](https://aka.ms/vscode-remote/containers)**.
+
+#### GitHub Codespaces
+Follow these steps to open this workshop in a Codespace:
+1. Click the Code drop-down menu and select the **Open with Codespaces** option.
+2. Select **+ New codespace** at the bottom on the pane.
+
+For more info, check out the [GitHub documentation](https://docs.github.com/en/free-pro-team@latest/github/developing-online-with-codespaces/creating-a-codespace#creating-a-codespace?WT.mc_id=academic-59300-cacaste).
+
+#### Binder
+This workshop is also available on Binder. To open the notebook in a Binder environment, just click the button below.
+
+[![Binder](https://mybinder.org/badge_logo.svg)](https://mybinder.org/v2/gh/carlotta94c/workshop-library/introToClusteringR%26Tidymodels/?labpath=%2Ffull%2Fintro-clustering-R-tidymodels%2Fsolution%2FChallenge-Clustering.ipynb)
+
+#### Learn Sandbox
+You can go through this challenge also leveraging on the Learn Sandbox environment, provided by [Unit 7](https://docs.microsoft.com/en-us/learn/modules/introduction-clustering-models/7-challenge-clustering?WT.mc_id=academic-59300-cacaste) of the MS Learn module - Introduction to clustering models by using R and tidymodels. Just sign in with your Microsoft or GitHub account and click on **Activate sandbox** to start.
+
+#### VS Code Remote - Containers
+Follow these steps to open this workshop in a container using the VS Code Remote - Containers extension:
+
+1. If this is your first time using a development container, please ensure your system meets the pre-reqs (i.e. have Docker installed) in the [getting started steps](https://aka.ms/vscode-remote/containers/getting-started?WT.mc_id=academic-59300-cacaste).
+
+2. Press <kbd>F1</kbd> select and **Add Development Container Configuration Files...** command for **Remote-Containers** or **Codespaces**.
+
+   > **Note:** If needed, you can drag-and-drop the `.devcontainer` folder from this sub-folder in a locally cloned copy of this repository into the VS Code file explorer instead of using the command.
+
+3. Select this definition. You may also need to select **Show All Definitions...** for it to appear.
```

Comment:
What does this  mean by _"this definition"_ ? Which one is the definition I would need to select?